plugins/redis/redis.cpp

```diff
@@ -690,10 +690,10 @@ void Connection::encodeChannel(StringBuffer & channel, const char * key) const
 {
     channel.append(REDIS_LOCK_PREFIX).append("_").append(key).append("_").append(database);
 }
-bool Connection::lock(ICodeContext * ctx, const char * key, const char * channel)
+bool Connection::lock(ICodeContext * ctx, const char * key, const char * channel, unsigned expire)
 {
     StringBuffer cmd("SET %b %b NX EX ");
-    cmd.append(timeout/1000);
+    cmd.append(expire);
```

Comment:
This change will conflict with that in the 5.2.8 upmerge. This version needs to remain.